static/app/views/alerts/rules/metric/ruleConditionsForm.tsx

```diff
@@ -436,26 +436,29 @@ class RuleConditionsForm extends PureComponent<Props, State> {
     const eventOmitTags =
       dataset === 'events' ? [...measurementTags, ...transactionTags] : [];
 
+    const hasMetricDataset =
+      organization.features.includes('metrics-performance-alerts') ||
+      organization.features.includes('mep-rollout-flag');
+
     return (
       <Fragment>
         <ChartPanel>
           <StyledPanelBody>{this.props.thresholdChart}</StyledPanelBody>
         </ChartPanel>
-        {showMEPAlertBanner &&
-          organization.features.includes('metrics-performance-alerts') && (
-            <AlertContainer>
-              <Alert type="info" showIcon>
-                {tct(
-                  'Filtering by these conditions automatically switch you to indexed events. [link:Learn more].',
-                  {
-                    link: (
-                      <ExternalLink href="https://docs.sentry.io/product/sentry-basics/search/" />
-                    ),
-                  }
-                )}
-              </Alert>
-            </AlertContainer>
-          )}
+        {showMEPAlertBanner && hasMetricDataset && (
+          <AlertContainer>
+            <Alert type="info" showIcon>
+              {tct(
+                'Filtering by these conditions automatically switch you to indexed events. [link:Learn more].',
+                {
+                  link: (
+                    <ExternalLink href="https://docs.sentry.io/product/sentry-basics/search/" />
```

Comment:
unrelated; is this just a placeholder link for now? 🤔

Reply:
hmm the copy isn't great either

Reply:
@scttcper The copy we use in dashboards is
> Your selection is only applicable to stored event data. We've automatically adjusted your results.

no docs mention too